microservices-connector/cmd/router/main.go

```diff
@@ -198,6 +199,40 @@ func executeStep(
 	return callService(step, serviceURL, input, headers)
 }
 
+func mergeRequests(initReq, respReq []byte) []byte {
```

Comment:
there should be a place to add some predefined parameter value like https://github.com/opea-project/GenAIComps/blob/main/comps/cores/proto/docarray.py#L92

Reply:
Hi @irisdingbj, this is defined as `parameters` for all example, I have defined it in const.